prime-dotnet-webapi/Services/Rules/AutomaticAdjudicationRules.cs

```diff
@@ -108,30 +109,45 @@ record = await _collegeLicenceClient.GetCollegeRecordAsync(cert.Prefix, cert.Lic
                 if (record == null)
                 {
                     await _businessEventService.CreatePharmanetApiCallEventAsync(enrollee.Id, cert.Prefix, cert.LicenseNumber, "Record not found in Pharmanet.");
-                    if (cert.NonPrescribingPrefix != null)
+                }
+                else
+                {
+                    await _businessEventService.CreatePharmanetApiCallEventAsync(enrollee.Id, cert.Prefix, cert.LicenseNumber,
+                        $"A record was found in Pharmanet with effective date {record.EffectiveDate:dd MMM yyy}.");
```

Comment:
"Pharmanet" should have proper casing "PharmaNet" (https://www2.gov.bc.ca/gov/content/health/health-drug-coverage/pharmacare-for-bc-residents/pharmanet)